mmv1/products/gkebackup/RestorePlan.yaml

```diff
@@ -496,6 +547,87 @@ properties:
                     description: |
                       A string that specifies the desired value in string format
                       to use for transformation.
+      - !ruby/object:Api::Type::Array
+        name: volumeDataRestorePolicyBindings
+        description: |
+          A table that binds volumes by their scope to a restore policy. Bindings
+          must have a unique scope. Any volumes not scoped in the bindings are
+          subject to the policy defined in volume_data_restore_policy.
+        item_type: !ruby/object:Api::Type::NestedObject
+          required: true
+          properties:
+            - !ruby/object:Api::Type::Enum
+              name: policy
+              required: true
+              description: |
+                Specifies the mechanism to be used to restore this volume data.
+                See https://cloud.google.com/kubernetes-engine/docs/add-on/backup-for-gke/reference/rest/v1/RestoreConfig#VolumeDataRestorePolicy
+                for more information on each policy option.
+              values:
+                - :RESTORE_VOLUME_DATA_FROM_BACKUP
+                - :REUSE_VOLUME_HANDLE_FROM_BACKUP
+                - :NO_VOLUME_DATA_RESTORATION
+            - !ruby/object:Api::Type::Enum
+              name: volumeType
+              required: true
+              description: |
+                The volume type, as determined by the PVC's
+                bound PV, to apply the policy to.
+              exactly_one_of:
+                - restoreConfig.0.volumeType
+              values:
+                - :GCE_PERSISTENT_DISK
+      - !ruby/object:Api::Type::NestedObject
+        name: restoreOrder
+        description: |
+          It contains custom ordering to use on a Restore.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: groupKindDependencies
+            required: true
+            description: |
+              A list of group kind dependency pairs
+              that is used by Backup for GKE to
+              generate a group kind restore order.
+            item_type: !ruby/object:Api::Type::NestedObject
+              required: true
+              properties:
+                - !ruby/object:Api::Type::NestedObject
+                  name: satisfying
+                  required: true
+                  description: |
+                    The satisfying group kind must be restored first
+                    in order to satisfy the dependency.
+                  properties:
+                    - !ruby/object:Api::Type::String
```

Comment:
If `resourceGroup` is not specified in the configuration and not included in the request to API, will it default to core group in the API side in the response?

Reply:
We can leave this empty. Internally in the backend we would infer empty as a core group so it does not need to have any resourceGroup value for it.